src/main/webapp/app/entities/lecture/lecture.model.ts

```diff
@@ -14,5 +14,8 @@ export class Lecture implements BaseEntity {
     studentQuestions: StudentQuestion[];
     course: Course;
 
+    // helper attribute
+    isAtLeastInstructor: boolean;
```

Comment:
I don't like storing this attribute in the lecture model! I think this should be moved to `lecture.component.ts`.

Reply:
I just discussed with @krusche: As we use this approach in other places as well, we will leave it like this for now.